src/screens/claimables/ClaimingSponsoredClaimable.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { useState } from 'react';
+import { SponsoredClaimable } from '@/resources/addys/claimables/types';
+import { ClaimingClaimableSharedUI, ClaimStatus } from './ClaimingClaimableSharedUI';
+
+export const ClaimingSponsoredClaimable = ({ claimable }: { claimable: SponsoredClaimable }) => {
+  const [claimStatus, setClaimStatus] = useState<ClaimStatus>('idle');
+
+  return <ClaimingClaimableSharedUI claim={() => {}} claimable={claimable} claimStatus={claimStatus} setClaimStatus={setClaimStatus} />;
```

Comment:
instead of passing an empty inline arrow function here can `claim` be optional instead?

Reply:
this is just temporary, in the next PR i pass a function that is required

Reply:
yeah my bad, trying to minimize the confusion without deleting too much code that was already written before breaking up my original branch